### DISCUSSION TITLE
Welcome to Cloudforet Mentorship Program

### DISCUSSION THREAD

**Discussion by choonho** [cloudforet-io/community#132] [Mentorship]
# Welcome to Community

## Program

https://mentorship.lfx.linuxfoundation.org/project/2906237a-5d0f-48b4-90a1-cb8ad0fd0d25

## Schedule

https://github.com/cloudforet-io/tsc/tree/master/mentorship

## I Love Community

![image](https://github.com/cloudforet-io/cloudforet/assets/3993088/eaa91fe3-4de4-4f68-9831-4929918d058f)

![image](https://github.com/cloudforet-io/cloudforet/assets/3993088/d37323db-96a0-4c7d-b5ff-9dedb3a0e4ff)

**Comment by choonho** [comment]
Hello All, 

The mentoring program is set to start on November 26th, and there is a chat channel of getting to know each other beforehand. 

# Join Community Channel 
https://github.com/orgs/cloudforet-io/discussions/categories/mentorship 

Please connect to the community channel and introduce yourselves while getting to know each other. As a Cloudforet contributor, please "follow" on the Cloudforet project and "star" both Cloudforet and artwork. 

# Survey 
https://docs.google.com/forms/d/e/1FAIpQLSfkmbIPuPc5AyN2lDcBCmhxjD2lt_W-YS_Llatxo40MfXSdxA/viewform 
Before we begin, we will conduct a brief survey. We kindly request your active participation. Best regards.

**Comment by ATREAY** [comment]
Hey everyone,

I am atreay kukanur, an enthusiast deeply passionate about open-source technology and cloud-native solutions. Recently, I completed the zero-to-merge cohort-1 program organized by CNCF and am currently an active contributor to the CNCF TAGENV blog. In addition, I actively contribute to projects spanning different domains such as blockchain, MERN Stack, and design.

I am honored and eager to be a part of this mentorship program. I am looking forward to engaging with esteemed professionals and like-minded individuals within this platform. I am excited about the learning and growth opportunities that lie ahead. 

Thank you for having me, and can't wait to dive in and get the ball rolling!

Let's connect on:
[LinkedIn](https://www.linkedin.com/in/atreay-kukanur/)
[Twitter](https://twitter.com/AtreayKukanur)

**Reply by choonho** [reply]
Hello Atreay,
Welcome to Woniverse!!!

**Comment by arup1221** [comment]
Hello Everyone,
   I'm Arup Gope, a third-year undergraduate student pursuing a Bachelor of Engineering. I'm a passionate developer with a focus on Python, Django. My technical skills include NextJS, ReactJS, Golang, NodeJS, MongoDB, Docker and more. I'm an avid learner, tech enthusiast, and community participant. I'm eager to contribute to the open source projects, and I believe the LFX mentorship program is an excellent way to achieve that goal.
   
   Thank you for this opportunity and I appreciate this opportunity and have full confidence in my ability to bring valuable contributions to the organization. I'm ready to seize this chance to learn, grow, and make a meaningful impact.

**Reply by choonho** [reply]
LFX mentorship is good starting point for open source contribution. I hope you will achieve your goal!

**Comment by AKACHI-4** [comment]
Hey, mates! 👋

I am Adarsh, a developer who loves to build innovative projects. I have a strong grasp of web technologies and cloud computing, and I am continuously on an endless learning journey with them. Communities and their projects play a significant role in my interests, and I've been contributing to open-source projects for the past 2 years. he past 2 years. 

In addition to this experience, I'm eager to contribute to larger organization projects, and I believe the LFX mentorship program is an excellent way to achieve that goal. 🚀

Thank you for this opportunity. I am confident that I can make meaningful contributions to the organization. 💚

**Reply by choonho** [reply]
Hi Adarsh! 🚀 That's awesome to hear about your passion for web technologies, cloud computing, and open-source contributions. Best of luck with the LFX mentorship program, and I'm sure you'll make meaningful contributions to larger organization projects. 💪🌟

**Comment by justankiit** [comment]
Hey Everyone,
I’m an Open-Source enthusiast and also a junior pursuing my Bachelor’s in Computer Science and engineering. I am passionate about Kubernetes, Web Dev, Artificial Intelligence, and Cloud/DevOps & I enjoy learning new things.
Regarding my past experiences, I work as a Web developer at GrowInCommunity, Also, I am a Mentee and Approver at CNCF Glossary in Hindi Localization. I am actively contributing to various Open-source projects of CNCF(Majorly Kubernetes, Layer5). I love to participate in Open-Source Programs(GSOC, SSOC) and mentoring in these programs. Mentored 50+ Students worldwide. I have also given talks and attended Tech Events at various national and International conferences and have conducted hands-on workshops on DevOps, Development, etc. I am also a big fan of events like KubeCon, and Open-Source Summit and have been attending for the past 2 years. I enjoy meeting and collaborating with people and working on real-world projects to enhance my skills as well as help others along the way.

**Reply by choonho** [reply]
Your extensive experience, contributions, and mentoring in various open-source programs, as well as your participation in tech events, are impressive. Keep up the great work in your journey of learning and collaboration! 👏🌟

**Comment by py-dev-nandini-12** [comment]
Hi Everyone,
I'm  Nandini a passionate and dedicated individual making a significant career change from being a physics teacher to pursuing my dream of becoming a professional Python developer. With a foundation in Python programming and hands-on experience in web development using Django, Flask, and RESTful APIs, I am excited to apply my skills to real-world commercial projects.I am enthusiastic about transitioning my passion for coding into a rewarding career as a Python developer. Eager to contribute to commercial projects, I am committed to continuous learning and staying updated with the latest industry trends.I am thrilled about the prospect of collaborating with industry professionals, contributing my skills to innovative projects, and further enhancing my expertise. I am open to opportunities that will allow me to grow, learn, and make a meaningful impact in the field of software development.
Thank you for considering my profile, and I look forward to the exciting challenges and opportunities that lie ahead in this fulfilling journey.

**Reply by choonho** [reply]
Hello Nandini! Your determination to transition from being a physics teacher to pursuing a career as a Python developer is inspiring. Your foundation in Python programming and web development, along with your commitment to continuous learning, will serve you well in your journey. Best of luck with your career change, and I hope you find fulfilling opportunities in software development! 🚀👩‍💻

**Comment by DivvSaxena** [comment]
Hey Folks!
I'm a passionate learner skilled in design, development, and community management, with a deep fascination for the internet. I am also a developer who specializes in building really exciting projects.
Thanks for considering my profile , looking forward to meet fellow folks and collaborate on building something interesting!

**Reply by choonho** [reply]
It's always great to meet fellow enthusiasts, and I'm sure you'll have exciting collaborations ahead. Here's to building something interesting together! 🚀👋

**Comment by tsathish27** [comment]
Hello everyone!

I'm Sathish, a passionate frontend developer currently pursuing my bachelor's degree in Computer Science & Engineering. I'm thrilled to be a part of this community and I'm eager to collaborate, contribute, and connect with new friends who share the same interests. Thank you for considering me, and I'm looking forward to this exciting journey ahead!

**Reply by choonho** [reply]
Welcome to the journey, and I hope you have a fantastic time here! 🚀👋

**Comment by 0xPunkItachi** [comment]
hello everyone My name is Yash, I am 2nd year Btech student in Electronics and communication branch. Right now i getting my hands dirty in devops, blockchain & web dev stuffs, building projects etc.  So Looking forward to that here I am to connect with good folks and learn from them & make some good open source contribution .

**Reply by choonho** [reply]
Hi Yash! It's great to hear about your diverse interests, from DevOps and blockchain to web development. Connecting with others, learning, and making open-source contributions are excellent ways to further your skills and knowledge. Welcome, and I hope you have a rewarding experience here connecting with like-minded individuals and working on exciting projects! 🚀👋

**Comment by Cvr421** [comment]
Hello Everyone I am Chandravijay Rai From India . I am a Full stack and Kubernetes application Developer and DevOps Enthusiast . From last 10 month working with the Open Source community . Thanks for this opportunity . I am very exited to work with you guys and make some Impactful contribution.

**Reply by choonho** [reply]
Hello Chandravijay! Your experience in full stack development, Kubernetes, and your enthusiasm for DevOps is fantastic. Open source contributions are a great way to make an impactful difference, and I'm sure you'll have a valuable journey working with the community. Welcome, and best of luck with your contributions! 🚀👋

**Reply by Cvr421** [reply]
Thanks a lot @choonho great to have you as my mentor🚀🚀🚀

**Comment by AnsahMohammad** [comment]
Hello everyone,
My name is Ansah and I am a third-year undergraduate student majoring in Computer Science and Engineering at NIT Surat. 
I am passionate about open source and collaboration. I enjoy solving real-world problems with technology. I have experience in developing backend applications using Django, working with quantum computers, and creating deep learning models. I also have a strong interest in core computer science concepts such as computer networks and operating systems.

I really appreciate this opportunity and I am eager to learn a lot and make valuable contributions to the organization.

**Reply by choonho** [reply]
Hello Ansah! Your passion for open source, collaboration, and your diverse skills in backend development, quantum computing, and deep learning is impressive. Your interest in core computer science concepts is fantastic too. I'm sure you'll make valuable contributions to the organization and continue learning and growing in your journey. Best of luck, and enjoy the experience! 🚀👏

**Reply by AnsahMohammad** [reply]
Hello @choonho , I'm excited about this LFX mentorship. I would like to learn more about how this works. 
For example, will we be assigned issues individually or in groups? Will there be any learning materials or resources available?

**Comment by sarniu** [comment]
Hello everyone,
I'm Pankaja from India. I'm graduated recently and currently a software engineer. I'm very much excited for this mentorship to begin as this is my first ever open source experience and looking forward to learn more and contribute in any way I can and have a positive impact :)

**Reply by choonho** [reply]
Welcome to open source! Your enthusiasm is great! As a software engineer, your skills are valuable. Dive in, learn, and contribute – you'll make a positive impact. If you need help, ask. Best of luck! 😊🚀

**Comment by Ayush-Sharma410** [comment]
Hey eveyone,
I am Ayush from India. I am currently in  I am a third-year undergraduate student majoring in Computer Science and Engineering at Thapar University. I've been into data science, machine learning and coding for the past 2 years now. I have an year of experience in web scraping and have some internship experience in this domain .I have recently completed my first Hacktoberfest and done some contributions to open source. I look forward to to this mentorship and learn some new and amazing skills on the way and making some great connections. Really excited to work on the amazing open source projects coming my way.

**Reply by choonho** [reply]
Best of luck with your journey, and I'm sure you'll excel in your open source endeavors and continue to grow your skills and network. Enjoy the ride! 😊🚀

**Reply by Ayush-Sharma410** [reply]
Thanks a lot @choonho. It feels great to have you as my mentor.😊

**Comment by cui36** [comment]
Hello, everyone! I'm Bruce from China, currently taking a gap year after completing my Bachelor's degree. During this time, I've had the opportunity to delve into the fascinating world of Cloud Computing through my personal research. I'm genuinely enthusiastic about making meaningful contributions in this field.
This marks my very first foray into the realm of open source, and I am fully committed to immersing myself in this experience. Let's collaborate, learn, and strive towards our collective goals, all while nurturing valuable connections for the future!

**Reply by choonho** [reply]
Best of luck in your open source endeavors, and here's to building valuable connections and collective success! 😊🚀

**Reply by cui36** [reply]
Thank you @choonho !

**Comment by monalisha31** [comment]
Hey everyone,

I'm Monalisha from India, currently working as a research assistant. I'm eager to learn more about Cloudforet and become an active member of this community. Previously, I have worked as a mentee in google summer of code program and really enjoyed the workings of an open source projects. Therefore, I am eager to dive into mentorship program and meet new contributors.

**Reply by choonho** [reply]
Welcome to the Cloudforet community! Your background and previous experience in Google Summer of Code are impressive. Best of luck in your mentorship, and enjoy meeting new contributors! 😊🚀

**Comment by dheeraj-326** [comment]
Is resume upload stuck in this state for anyone else?

<img width="794" alt="image" src="https://github.com/cloudforet-io/cloudforet/assets/24998581/4051a23d-2e54-448e-a2a0-10950d6b2e45">

**Reply by choonho** [reply]
I am not sure. Maybe we closed the application. Try one more, if it is not work, don't worry. No problem.

**Comment by Tyagiquamar** [comment]
I am Mohd Quamar Tyagi from India. I am a 3rd year BE CS student. I'm eager to learn more about Cloudforet become an active member of this community and contribute as much as possible. I have contributed to open source since last year and am an active member and contributor of Layer5 Meshery. I have done one internship. This program will be helpful for me to learn and I am very excited to start my journey.
This project will be very beneficial for me as I want to become an open-source maintainer and become a mentor on LFX so that I can guide students in the future.

**Reply by choonho** [reply]
This program will undoubtedly provide you with the necessary tools and knowledge to achieve your goals. Best of luck with your journey, and I'm here to support you along the way.

**Comment by SammyBloom** [comment]
Hello everyone,

I am Samuel Nwanwobi from Nigeria, a Cloud DevOps Engineer with a strong foundation in software engineering. I have had the privilege of designing and managing cloud infrastructures, streamlining operations, and optimizing software delivery pipelines.

I'm thrilled to embark on a new journey by joining the Cloudforet Mentorship Program, marking my first venture into the open-source world. This exciting opportunity allows me to learn, collaborate, and contribute to the open-source community, all while building upon my cloud and DevOps expertise.

I look forward to the incredible learning experiences and the chance to work with passionate mentors and fellow enthusiasts in the Cloudforet program. Here's to the future of open source and my continued growth as a Cloud DevOps Engineer!

**Reply by choonho** [reply]
Samuel, your background in cloud DevOps engineering is impressive. Your experience in designing and managing cloud infrastructures and optimizing software delivery pipelines sets a strong foundation for your venture into the open-source world through the Cloudforet Mentorship Program. The opportunities for learning and collaboration in this program will undoubtedly fuel your growth as a professional in the field.

**Reply by SammyBloom** [reply]
Thank you so much for the privilege.

**Comment by sundarr786** [comment]
Hello Everyone,

I am Sundar, Currently Working in C language Domain.I am newbie to open source contribution.
I am excited to be part of the team and curious to learn and do things in mentorship program and contribute to the cloudforet.

Grateful for the opportunity provided.Nice to know you all.

**Reply by choonho** [reply]
Participating in this program will offer an excellent opportunity to gain valuable experience in open-source contributions.

**Comment by madguy02** [comment]
Hello Everyone,

I am Manish Kakoti, I am an opensource enthusiast. I have previously contributed to projects like Rocket.Chat and sequelize. I am looking foreward in contributing to cloudforet and also be a long term contributor to the project as well.

**Reply by choonho** [reply]
Great to have you here! Your previous contributions to projects like Rocket.Chat and sequelize demonstrate your dedication to the open-source community. With your enthusiasm and experience, your contributions to Cloudforet will undoubtedly be valuable. Looking forward to your long-term involvement and contributions to the project!

**Reply by madguy02** [reply]
Thanks a lot @choonho , looking forward to get things started

**Comment by Suraj-kumar00** [comment]
Hello everyone, how you all are doing?
My name is Suraj, 3rd year computer science student currently learning DevOps tools and technologies, and I have some experience in frontend development and currently working on my portfolio as well. So first thing I want to thank you for being selecting me in this open source project and I'll try to give my best in this project.

Thank you.

**Reply by choonho** [reply]
Hello, Suraj! It's great to hear from you. Congratulations on being selected for the open-source project! This will be a fantastic opportunity to contribute and learn. f you ever need any assistance or have questions along the way, feel free to reach out!

**Comment by MDMohaymenUlAnam** [comment]
Hello Everyone,
I am Mohaymen, 4th year software engineering student. I am learning cloud computing and Kubernetes: open source enthusiast and technical writer. Looking forward to learn more from this mentorship program with plugin development and hands on experience.

**Reply by Suraj-kumar00** [reply]
Hi Mohaymen great to hear that, I'm Suraj and I also have same interest can we connect?

**Reply by MDMohaymenUlAnam** [reply]
sure thing. Here is my linkedin profile - https://www.linkedin.com/in/mohaymen-ul-anam-215036246/

**Reply by choonho** [reply]
That's fantastic, Mohaymen! Your enthusiasm for cloud computing, Kubernetes, and open source is admirable. Pursuing plugin development and gaining hands-on experience sounds like a great next step for you. I'm sure the mentorship program will provide valuable insights and opportunities for your growth in these areas. Best of luck on your journey!

**Comment by achiverram28** [comment]
I am Ram Samarth B B a 3rd year Computer Science Undergrad, a huge Cloud Native and DevOps Enthusiast. I always love playing around with Kubernetes as feel it is one of the love that I have found it in my DevOps journey . I have built my own Kubernetes Stack Client in GoLang.
I have successfuly completed one year internship as a Backend Developer and a DevOps Engineer at a UK based startup where I have handled the core backend and admin operation using typescript . I also have taken care of the Cloud Deployments and Storage in the GCP.
As far as Python is concerned I am a baseline developer and a Research Intern at Federated Learning Labs where I have contributed an entire federated learning paper code in the Flower FL Framework .

So I love this field where I can learn to always keep my applications at scale , while also applying this at my distributed AI research.

I always love to take LFX Mentorship opportunities and would like to contribute widely to Open Source

**Reply by choonho** [reply]
Wow, Ram! Your passion for Cloud Native technologies, DevOps, and Kubernetes shines through your experiences. Wishing you continued success in your LFX Mentorship and contributions to Open Source!

**Reply by achiverram28** [reply]
Thanks for this .
Can we get an insight as to who all are been selected for this ?

**Reply by achiverram28** [reply]
Like all of guys present here are mentees or any selection criteria , and same questionas that of @madguy02

**Reply by choonho** [reply]
Everyone in here is mentees. This program is the beginner guide for Cloudforet Plugin Development. After complete this program, there will be advanced program, who will become comitter.

**Comment by madguy02** [comment]
@choonho what would be the next step for us? as we approach closer to the starting date.

**Reply by choonho** [reply]
Hello All,
The schedule for Mentorship program is 
[Mentorship Program Schedule](https://github.com/cloudforet-io/tsc/tree/master/mentorship)

## Week 1. Orientation and Fundamentals
[Introduction to mentorship program](https://mentorship.lfx.linuxfoundation.org/project/2906237a-5d0f-48b4-90a1-cb8ad0fd0d25)
[Survey](https://docs.google.com/forms/d/e/1FAIpQLSfkmbIPuPc5AyN2lDcBCmhxjD2lt_W-YS_Llatxo40MfXSdxA/viewform?usp=sf_link)
[Introduction to Cloudforet](https://docs.google.com/presentation/d/1iJvxHBB79xADKaHt2HfjDVSwd1m_COxiURx_M2Vo1uA/edit?usp=sharing)
[Installation of Cloudforet with minikube](https://cloudforet.io/docs/setup_operation/quick_install/)

## Progress

[Progress](https://github.com/orgs/cloudforet-io/projects/7/views/1)

**Reply by madguy02** [reply]
@choonho one more question to add to this, does the mentees get added to cloudforet organization on github?

**Reply by choonho** [reply]
Mentee is not a member of organization. The person who has accepted as core or plugin committer will become a member.

The person who contribute a code will be listed as contributor list in a repository.

**Reply by madguy02** [reply]
Thanks a lot @choonho , looking forward to beginning the program

**Comment by SpringWiz11** [comment]
I am Kishan Gurumurthy, a third-year Computer Science undergraduate with a strong passion for Cloud Native and DevOps methodologies. My enthusiasm particularly extends to working with Kubernetes, a facet of my DevOps journey that I find profoundly rewarding.

In the course of my academic pursuits, I have developed a Kubernetes Stack Client in GoLang, showcasing my specialization in GoLang for backend and cloud development. Additionally, I bring practical experience from a year-long internship as a Software Engineering Intern at a UK-based startup. During this tenure, I managed core backend operations and administrative tasks using TypeScript, demonstrating my versatility. I also oversaw Cloud Deployments and Storage across platforms like GCP, AWS, and OpenStack.

In the realm of Python development, I am a proficient baseline developer and currently serve as a Research Intern at Federated Learning Labs. Here, I contributed significantly to a federated learning paper by coding the entire implementation in the Flower FL Framework. This experience has deepened my understanding and application of distributed AI research.

I am drawn to this field as it allows me to continually enhance my skills in maintaining applications at scale, a principle I actively apply to my ongoing distributed AI research.

I am eager to contribute and learn further within a mentoring relationship. Your guidance would be invaluable to my professional growth.

Thank you for considering my application.

Best Regards,

Kishan Gurumurthy

**Comment by Gaurav122000** [comment]
Hi all,
My name is Gaurav,
I'm currently pursuing my master's and for the past 5 yrs. I keep on learning new things throughout my course. I Have strong passion for Cloud Native, software development and computer networking. I have been developing my skills in programming languages or technologies like MERN Stack & AWS.
I have Created Many Projects in Web Development, and some projects in other domains. These experiences have taught me skills and new lessons.
I am excited about the opportunity to bring my skills and passion and contribute to the community project.
Thank you for considering me :)

**Comment by 0xPunkItachi** [comment]
Hey folks, hope you are doing well... I just wanted to ask that program is starting from today. so is there any zoom/meet or some schedule for today?

**Reply by madguy02** [reply]
i actually have the same question as well, cc: @choonho

**Comment by Cvr421** [comment]
Hello @choonho is there any schedule for today ?

**Reply by choonho** [reply]
Hello All, Mentorship program will start in this week. I will assign Task using LFX mentorship program that you can receive my e-mail. There will be no Zoom/meet.

**Reply by choonho** [reply]
Week 1. Agenda is Here, https://github.com/orgs/cloudforet-io/discussions/133

**Comment by Asfawh** [comment]
Hi everyone.
My name is Habtamu Asfaw.
I'm SRE and passionate about building robust, scalable, and secure systems that meet the needs of users and stakeholders. I have decade years of experience in designing, implementing, and managing production systems.
 
I love open-source projects, specially Cloud native and DevOps tools and I'm excited about the opportunity and eager to contribute to this community project.

**Comment by choonho** [comment]
# Tips

During mentoring, I will give your additional issues which can be commit with easy.

Issue Tag like "good first issue".

![image](https://github.com/cloudforet-io/cloudforet/assets/3993088/50f0ac31-d0df-4905-90eb-c42e036ec0f7)

Anyone who can fix, please make PR.

| Repository | Issue Number | Status |
| ---              | ---                    |    ---   | 
| docs          | https://github.com/cloudforet-io/docs/issues/169                 |  |
| docs          | https://github.com/cloudforet-io/docs/issues/171                |  |

**Comment by AKACHI-4** [comment]
Hey @choonho , aren't there any other available issues to work on that are currently open? 

I mostly have free time on weekends, so I'm searching for some. Also, I wanted to ask if, apart from installing and setting up Cloudforet, do we have to do anything else? 

Is contributing to an issue required for week-1? I missed both of them, and now they are closed. !!
